utils/sane-backends/files/saned.init

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh /etc/rc.common
+
+START=90
+STOP=10
+USE_PROCD=1
+
+sanedconf() {
+	local cfg="$1"
+	config_get user "$cfg" user nobody
+	config_get portrange "$cfg" portrange
+	config_get acl "$cfg" acl
+}
+
+start_service() {
+	local user hplip portrange acl
```

Comment:
On 16-03-25 12:03 AM, Luiz Angelo Daros de Luca wrote:
> In utils/sane-backends/files/saned.init
> <https://github.com/openwrt/packages/pull/2543#discussion_r57417628>:
>
>> @@ -0,0 +1,35 @@
>> +#!/bin/sh /etc/rc.common
>> +
>> +START=90
>> +STOP=10
>> +USE_PROCD=1
>> +
>> +sanedconf() {
>> +	local cfg="$1"
>> +	config_get user "$cfg" user nobody
>> +	config_get portrange "$cfg" portrange
>> +	config_get acl "$cfg" acl
>> +}
>> +
>> +start_service() {
>> +	local user hplip portrange acl
>
> hplip? there is no hplip package yet :-). In fact, I have it ready (you
> can check my personal github repo if you want).
> The updated hplip was split into a common, a cups and a saned package
> (which does not depend on cups). The problem is that cups (libcups) is
> still needed to build the package and it is still not available on
> openwrt/packages. @psycho-nico <https://github.com/psycho-nico> said he
> was working on it.

I've been working on this as well, but you're right hplip is not 
currently available.

Regards,

Daniel